tests/Common/TriggersTest.php

```diff
@@ -1079,11 +1077,26 @@ public function tokenOptionsProviderInvalid()
     public function deleteKeyProvider()
     {
         return [
-            'component' => ['component'],
-            'configurationId' => ['configurationId'],
-            'coolDownPeriodMinutes' => ['coolDownPeriodMinutes'],
-            'runWithTokenId' => ['runWithTokenId'],
-            'tableIds' => ['tableIds'],
+            'component' => [
+                'component',
+                'Invalid parameters - component: This field is missing.',
+            ],
+            'configurationId' => [
+                'configurationId',
+                'Invalid parameters - configurationId: This field is missing.',
+            ],
+            'coolDownPeriodMinutes' => [
```

Comment:
Nejsou tady špatně seřazené validátory? Čekal bych, že missing bude missing. A ne validátor na číslo...

Reply:
Už tam na to koukám, je to BC na "first error" OK